lib/managers/recent-items.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @fileoverview Manager for the Box RecentItem Resource
+ */
+
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+var urlPath = require('../util/url-path');
+
+// ------------------------------------------------------------------------------
+// Private
+// ------------------------------------------------------------------------------
+var BASE_PATH = '/recent_items';
+
+// ------------------------------------------------------------------------------
+// Public
+// ------------------------------------------------------------------------------
+
+/**
+ * Simple manager for interacting with all 'RecentItem' endpoints and actions.
+ *
+ * @constructor
+ * @param {BoxClient} client - The Box API Client that is responsible for making calls to the API
+ * @returns {void}
+ */
+function RecentItems(client) {
+	this.client = client;
+}
+
+/**
+ * Requests all items that have been accessed by a user in the last 90 days or the last 1000 items accessed.
+ *
+ * API Endpoint: '/recent_items'
+ * Method: GET
+ *
+ * @param {Object} [qs] - Additional options can be passed with the request via querystring. Can be left null in most cases.
```

Comment:
Could you please add documentation for the query string options documented at https://developer.box.com/reference#get-recent-items to the JSDoc here?  You can use the following format:

```
@param {string} [qs.marker] Marker field description goes here
```